### PR TITLE
Mobile QRscan page

### PR DESCRIPTION
**Pull request by firtap**

### CODE REVIEW

File: ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+using System.Windows.Input;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+using ZXing;
+
+namespace ScanShop.Mobile.ViewModels
+{
+    public class QRScanViewModel : BaseViewModel
+    {
+        public Command ScanCommand { get; set; }
```
Comment (Losdrew) [inline ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs:12]: It would be better to add newlines between fields, methods, constructors etc.

Comment (firtap) [reply ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs:12]: Fixed

**Review by Losdrew — APPROVED** [review]

File: ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            set
+            {
+                if (value == _QRScanText)
+                    return;
+                _QRScanText = value;
+                OnPropertyChanged();
+            }
+        }
+
+        private void ScanResultAction(object obj)
```
Comment (Wednumi) [inline ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs:33]: In my opinion, the method name "ProcessScanResult" or "HandleScanResult" would be better

Comment (firtap) [reply ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs]: Fixed

**Review by Wednumi — APPROVED** [review]

**Pull request merged**